packages/superset-ui-core/src/query/types/QueryFormData.ts

```diff
@@ -149,7 +154,7 @@ export interface DruidFormData extends BaseFormData {
   druid_time_origin?: string;
 }
 
-export type QueryFormData = SqlaFormData | DruidFormData;
+export type QueryFormData = DruidFormData | SqlaFormData;
```

Comment:
Switching order so if something doesn't add up, it will fallback to `SqlaFormData`